besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -4204,65 +4179,40 @@ public void quorumInteropEnabledFailsIfEnclaveKeyFileDoesNotExist() throws IOExc
     final Path genesisFile =
         createFakeGenesisFile(VALID_GENESIS_QUORUM_INTEROP_ENABLED_WITH_CHAINID);
     parseCommand(
-        "--goquorum-compatibility-enabled",
         "--genesis-file",
         genesisFile.toString(),
         "--min-gas-price",
         "0",
         "--privacy-public-key-file",
         "ThisFileDoesNotExist");
     assertThat(commandErrorOutput.toString())
-        .contains(
-            "--privacy-public-key-file must be set when --goquorum-compatibility-enabled is set to true.");
+        .contains("--privacy-public-key-file must be set when isquorum is set in genesis file.");
```

Comment:
I think this needs to match the other one `--min-gas-price must be set to zero if GoQuorum compatibility is enabled in the genesis config.`

Reply:
Done